content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -2,45 +2,97 @@
 
 In a regular Git workflow, <abbr>DVC repository</abbr> versions are typically
 synchronized among team members. And [DVC Experiments] are internally connected
-to this commit history. But to avoid cluttering everyone's copies of the repo,
-by default experiments will only exist in the local environment where they were
-[created].
+to this commit history, so you can similarly share them.
 
-You must explicitly save or share experiments individually on other locations.
-This is done similarly to [sharing regular project versions], by synchronizing
-with DVC and Git remotes. But DVC takes care of pushing and pulling to/from Git
-remotes in the case of experiments.
+## Basic workflow: store as peristent commits
+
+The most straightforward way to share experiments is to store them as
+[persistent](/doc/user-guide/experiment-management/persisting-experiments.md)
+Git commits and share them like you would [any other commit]. This will share
+all code and metadata associated with the experiment.
+
+```cli
+$ dvc exp branch exp-e6c97 my-branch
+Git branch 'my-branch' has been created from experiment 'exp-e6c97'.
+To switch to the new branch run:
+        git checkout my-branch
+
+$ git checkout my-branch
+Switched to branch 'my-branch'
+
+$ git push origin my-branch
+```
+
+If you only need to share code and metadata like parameters and metrics, then
+pushing to Git is often enough. However, you may also have data, models, etc.
+that are tracked and <abbr>cached</abbr> by DVC. If you need to share these
```

Comment:
Thanks for this contrib @dberenbaum and sorry for noticing it very late 😅 just a few Qs:

> how to share experiments without a DVC remote

Maybe we should have a separate section for experiments with no DVC-tracked data then?

Also, I'm not sure `exp branch/apply` + `git commit/checkout` + `git push` is more basic than `exp push`..?

And... doesn't it stop being an experiment once you commit it?

Reply:
> Maybe we should have a separate section for experiments with no DVC-tracked data then?

It's not only about experiments with no DVC-tracked data. Even if you have DVC-tracked data, it's not that common that you need to share it -- metrics, params, etc. are enough to see the results of the experiment and compare. In other words, this is probably a reasonable default, and having to share the actual DVC-tracked data may be the special case.

> Also, I'm not sure `exp branch/apply` + `git commit/checkout` + `git push` is more basic than `exp push`..?

👍 It's definitely more commands, which is why `exp push` exists. However, it's easier in that you don't need a DVC remote, and you are probably already familiar with those Git commands.

For context, this PR was to address sharing experiments generated by the new DVCLive experiments onboarding workflow. If you start using DVC that way, you have no idea about DVC-tracked data or remotes, so introducing all of this to share an experiment feels like overkill.  Maybe there's a better framing than "basic" vs. "advanced."

It's also possible to make `exp push` work better in this scenario and not have to introduce a separate workflow (see https://github.com/iterative/dvc/issues/8678). This was more of a stopgap so there is some info for now about how to share experiments in these simple scenarios, and to introduce the Git mechanisms behind sharing experiments.

> And... doesn't it stop being an experiment once you commit it?

IMHO this is more of a technicality than an important detail, but maybe I missed your point here.

Reply:
Good points @dberenbaum. And thanks for the context.

> you have no idea about DVC-tracked data or remotes, so introducing all of this to share an experiment feels like overkill. Maybe there's a better framing 

I still feel like what we call basic (whatever we call it, it's still on top) should not be emphasized as all other pages focus on what I thought was the primary UX here: `dvc exp`. My suggestion:

1. Start with `exp push/pull` because it's easier, most consistent UX, and [doesn't seem to require](https://github.com/iterative/dvc/issues/8678#issuecomment-1344542009) a DVC remote anyway.
  \* Do NOT link [to the GS](https://dvc.org/doc/start/data-management/data-versioning#storing-and-sharing) here (confusing, esp. since the other section also does).
  \* Just mention that IF you have a (default?) DVC remote, data will also be included (and you may want to avoid this with `--no-cache`). _p.s. should `--no-cache` [be the default](https://github.com/iterative/dvc/issues/8678#issuecomment-1396383315)?_

1. Move onto the more granular/manual option. It could be as simple as mentioning committing experiments to make them persistent (link to that guide), and that once they're regular project versions you can share normally (share to the right places). I do really like the ascii diagrams though, let's keep them unless there's a better place to move/copy one of them to...

Also:

- Prob. no need for `dvc remote` usage examples here (link).
- "Sharing many" -> "Sharing several" ? (Minor)

Reply:
Thanks @jorgeorpinel! PTAL also at https://www.notion.so/iterative/Sharing-Experiments-07b8d53f87b94b7291c289bb1eb45159, where there's more discussion happening about the product direction here.

Reply:
> dvc exp push is a heavyweight operation if people only want to share some metrics and plots... requires the same effort as "persisting" an experiment (push the full Git history and all DVC-tracked artifacts)...

That effort is all under the hood though. Until a better solution is implemented, `exp push/pull` is still an easier and more consistent UX, I think.

Reply:
> However, you get none of the benefits of a (persistent) Git commit, since the Git forge ... (does not show) pushed experiments.

I guess this is the main concern, which I get. But again, we're already in the context of the special construct that the _DVC experiment_ is and we're pretty clear about their limitations. Examples from https://dvc.org/doc/user-guide/experiment-management:

_Experiments preserve a connection to ... (Git HEAD) as their parent or baseline, but **do not form part of the regular Git tree** (unless you make them [persistent](https://dvc.org/doc/user-guide/experiment-management/persisting-experiments)).
Review and compare experiments **in a GUI (Studio, VSC) or from the command line**_

There's also an entire [Reviewing and Comparing Experiments](https://dvc.org/doc/user-guide/experiment-management/comparing-experiments) doc (prob. needs rewrite but still) detailing where/how to show and compare them.

Reply:
Anyway... PTAL at [my proposal](https://dvc-org-guide-exps-shar-3ai4hw.herokuapp.com/doc/user-guide/experiment-management/sharing-experiments) @dberenbaum . It simplifies and reorders the sections, being explicit about pros and cons. PR: #4271

Reply:
Thanks @jorgeorpinel. I didn't mean to ignore your comments. I've just not gotten a chance to revisit this and wasn't making it a priority because revamping experiment sharing is one of this quarter's product directions, so I expect this will change again soon. Do you think it's bad enough that we need to do an iteration immediately?

Reply:
Another thought here: in the spirit of docs-driven development, we could work on #4271 as part of the initial product development and try to sync it with our plans. What do you think?

Reply:
Sounds good! Thanks